### PR TITLE
[GH-7] go modules instead of go dep

### PR DESCRIPTION
**Pull request by 030**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github/030/golang-bitbucket-cloud-build-status-notifier
+
+go 1.12
```
[the remark this answers is not in the document]
Comment (jorianvo) [reply go.mod]: So the steps are:

1. Run `go mod init <module name>`
1. Run `go build` for the first time to resolve all dependencies and populate the `go.mod` and `go.sum`. 
1. Commit these files

[the remark this answers is not in the document]
Comment (030) [reply go.mod]: Ok. I will do that. `go mod download` in the pipeline would be sufficient once the sum and go.mod have been committed?

[the remark this answers is not in the document]
Comment (jorianvo) [reply go.mod]: You don't need the `go mod download` at all. The `go build` step is sufficient.

[the remark this answers is not in the document]
Comment (030) [reply go.mod]: Ok. Go.mod and go.sum have been committed. The pipeline will be adjusted as well.

[the remark this answers is not in the document]
Comment (030) [reply go.mod]: I have processed the feedback. Could you review again?

**Comment by 030** [comment]
@jorianvo feedback has been processed. Could you review it again?

**Pull request merged**